File: dataset/output/CLAUDE_claude-3-5-sonnet-20240620/Task157_CLAUDE_claude-3-5-sonnet-20240620.java

public class MemoryAllocation {
    public static void main(String[] args) {
        // Allocate memory
        int[] array = new int[10];

        // Use the allocated memory
        array[0] = 5;
        System.out.println(array[0]);

        // Memory is automatically deallocated when array goes out of scope
    }
}

File: dataset/output/CLAUDE_claude-3-5-sonnet-20240620/Task157_CLAUDE_claude-3-5-sonnet-20240620.cpp

#include <iostream>

int main() {
    // Allocate memory
    int* ptr = new int[10];

    // Use the memory
    ptr[0] = 5;
    std::cout << ptr[0] << std::endl;

    // Deallocate memory
    delete[] ptr;

    return 0;
}
